engines/router/configs/custom_ensembling_router.yaml

```diff
@@ -3,7 +3,6 @@ id: combiner
 fan_in:
   type: fiber.EnsemblingFanIn
   properties:
-    default_route_id: route_id_1
```

Comment:
No harm having this in the router example but this value will not be set by the API anymore, for custom ensemblers.